Modules/RosterManager/Options.lua

```diff
@@ -448,9 +443,31 @@ local function dynamic_item_values(roster, equationGet, equationSet, multiplierG
         set = (function(i, v) multiplierSet(tonumber(v)) end),
         name = CLM.L["Multiplier"],
         pattern = CONSTANTS.REGEXP_FLOAT,
+    }
+	order = order + 1
+    args["equation_expvar"] = {
+        type = "input",
+        desc = CLM.L["Exponential scaling value used by the equations; Base for EPGPWeb, or Exponent for WoWpedia"],
```

Comment:
This is a bit confusing description as it suggests 3 uses while in reality its 2